extensions/wikia/ArticleVideo/scripts/jwplayer-instant.js

```diff
@@ -0,0 +1,75 @@
+(function () {
+	var videoDetails = window.wgFeaturedVideoData;
+	if (!videoDetails) {
+		return;
+	}
+	var videoElementId = 'featured-video__player';
+	var videoId = videoDetails.videoId;
+	var playerInstance = jwplayer(videoElementId);
+	window.featureVideoPlayerInstance = playerInstance;
```

Comment:
globals are bad :)

maybe a module with it so we could require it in both places?